editor/src/uuiui/menu-tab.tsx

```diff
@@ -32,16 +32,15 @@ export const MenuTab: React.FunctionComponent<React.PropsWithChildren<MenuTabPro
     const label = defaultIfNull<React.ReactElement | string>('', props.label)
 
     const baseStyle = {
-      paddingLeft: 4,
-      paddingRight: 4,
-      paddingBottom: 6,
+      padding: '4px 6px',
```

Comment:
the asymmetric styling top vs bottom was intentional to account for the inset box shadow and still visually centre the text - pls check you're happy with the result everywhere we use the tabs in the editor